### PR TITLE
Variation tracks for Human CHM13

### PR DESCRIPTION
**Pull request by Gurpreet-Ghattaoraya**
Adding variation tracks for human T2T-CHM13.

Example of working tracks are available on this sandbox link: http://wp-np2-25.ebi.ac.uk:8001/Homo_sapiens_GCA_009914755.4/Location/View?db=core;r=1:375270-375521;v=rs78067484;vdb=variation;vf=7:56179501:A_G:gnomADg

### CODE REVIEW

**Review by jamie-m-a — CHANGES_REQUESTED** [review]
Needs updating with the more complete GWAS id and updated descriptions for both gwas and dbsnp tracks

**Review by jamie-m-a — APPROVED** [review]
Looks good!

File: conf/json/homo_sapiens_gca009914755v4_vcf.json [hunk 1]
```diff
@@ -73,7 +73,28 @@
...
+    {
+      "id": "NHGRI-EBI_GWAS_catalog",
+      "species": "Homo_sapiens_GCA_009914755.4",
+      "assembly": "T2T-CHM13v2.0",
+      "description": "GWAS Catalog data was generated by our colleagues in Telomere-to-telomere consortium CHM13 project by intersecting RefSeq IDs in the GWAS Catalog associations v1.0 file with all dbSNP build 155 variants. These were lifted over from GRCh38 to the T2T-CHM13 assembly.",
+      "type": "local",
+      "use_as_source" : 1,
+      "use_seq_region_synonyms": 0,
+      "use_vcf_consequences": 1,
+      "filename_template" : "/nfs/services/nobackup/ensweb/variation-pgs/CHM13/Homo_sapiens-GCA_009914755.4-2023_01-GWAS.vcf.gz"
```
Comment (jyothishnt) [inline conf/json/homo_sapiens_gca009914755v4_vcf.json:97]: Same here

File: conf/json/homo_sapiens_gca009914755v4_vcf.json [hunk 2]
```diff
@@ -73,7 +73,28 @@
...
+    {
+      "id": "dbSNP155",
+      "species": "Homo_sapiens_GCA_009914755.4",
+      "assembly": "T2T-CHM13v2.0",
+      "description": "dbSNP build155 variants, lifted over from GRCh38 to T2T-CHM13 by colleagues in the Telomere-to-telomere consortium CHM13 project.",
+      "type": "local",
+      "use_as_source" : 1,
+      "use_seq_region_synonyms": 0,
+      "use_vcf_consequences": 1,
+      "filename_template" : "/nfs/services/nobackup/ensweb/variation-pgs/CHM13/Homo_sapiens-GCA_009914755.4-2023_01-dbSNP155.vcf.gz"
```
[the remark this answers is not in the document]
Comment (Gurpreet-Ghattaoraya) [reply conf/json/homo_sapiens_gca009914755v4_vcf.json]: Changed to point to the FTP path instead.

**Review by jamie-m-a — COMMENTED** [review]
I'm slightly confused because the clinvar and gnomad files are shwoing as an /nfs location and the 2 new tracks as an https link?

Shouldn't these be consistent? @jyothishnt can you comment?

It might be that we need to wait for info from Likhitha on monday as I think she had this set up with Anne as to where the files should go.

**Review by jamie-m-a — COMMENTED** [review]
Looks correct @jyothishnt I can't view files in that dir - can you confirm they are present - thanks!

**Comment by Gurpreet-Ghattaoraya** [comment]
Closing this PR and reopening as fresh. This PR has become messy.

**Pull request closed without merging**